resources/i18n/pt.json

```diff
@@ -1563,14 +1563,11 @@
     "CloneAsFolder": "Clonar como Pasta",
     "CloneSuccess": "O pedido de clone foi enviado com sucesso.",
     "category": "Categoria",
-    "CloneInfo": "__NOT_TRANSLATED__"
+    "CloneInfo": "Ele será clonado como sua pasta de tipo de usuário."
   },
```

Comment:
@yomybaby 
I found some parts that hasn't been translated yet, so I replaced with auto-translation.